api/v1alpha1/dynakube_types.go

```diff
@@ -61,6 +63,12 @@ type DynaKubeSpec struct {
 
 	ClassicFullStack FullStackSpec `json:"classicFullStack,omitempty"`
 
+	// Enables Routing
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="Routing"
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:text"
```

Comment:
operator-sdk generation comments changed.
Please have a look at https://github.com/Dynatrace/dynatrace-operator/pull/66/files#diff-4579debe2858b0359e8be4974e780df4162f3d4ed96e8f95bb34d62cd365dc9a for examples

Reply:
updated comments